### PR TITLE
Ipv6 template

### PR DESCRIPTION
**Pull request by EricTheRed999**
Adding a second IPv6 VNET sample - this one uses Standard Load Balancer (first sample was Basic LB)

### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [ EL_CHECKED ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: ipv6-in-vnet-StdLB/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Create a dual stack IPv4/IPv6 VNET with 2 VMs
```
Comment (bmoore-msft) [inline ipv6-in-vnet-StdLB/README.md]: add a note here that mentions the feature must be registered in the sub before you can deploy the template

File: ipv6-in-vnet-StdLB/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+        "platformFaultDomainCount": "2",
+        "platformUpdateDomainCount": "5"
+      },
+      "sku": {
+        "name": "Aligned"
+      }
+    },
+    {
+      "apiVersion": "2019-04-01",
+      "type": "Microsoft.Storage/storageAccounts",
```
Comment (bmoore-msft) [inline ipv6-in-vnet-StdLB/azuredeploy.json]: this is never used

Comment (EricTheRed999) [reply ipv6-in-vnet-StdLB/azuredeploy.json]: Brian, please explain...what exactly is never used?

Comment (bmoore-msft) [reply ipv6-in-vnet-StdLB/azuredeploy.json]: The storage account - we're creating one in the template, but it's not used for anything...

**Pull request merged**